includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -292,6 +292,16 @@ public function __construct(
 			// no longer valid options.
 			unset( $this->form_fields['payment_request_button_type']['options']['branded'] );
 			unset( $this->form_fields['payment_request_button_type']['options']['custom'] );
+
+			add_filter(
```

Comment:
Moved from outside the "grouped settings" check to inside of it.
Converted to individual methods (so that they could be overridden in UPE)